extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -474,8 +475,7 @@ export class AmpA4A extends AMP.BaseElement {
           if (!creativeParts || !creativeParts.signature) {
             return Promise.resolve();
           }
-          this.protectedEmitLifecycleEvent_(
-              'adResponseValidateStart', creativeParts);
+          this.protectedEmitLifecycleEvent_('adResponseValidateStart');
```

Comment:
validateStart/End will include the time it takes to possibly fetch the public keys.  Ideally we would have an explicit timing around native crypto call as well.  In addition, we could consider including additional info such as the number of networks verified and whether it did indeed pass validation (although we could infer that if friendly frame render is used although its possible layoutCallback may never be called)